src/corehost/cli/fx_reference.h

```diff
@@ -47,7 +47,7 @@ class fx_reference_t
         return fx_version_number;
     }
 
-    const bool get_apply_patches() const
+    bool get_apply_patches() const
```

Comment:
This isn't an API/ABI break, is it?

Reply:
These are not external APIs, so as long as the compiler puts it all together correctly - it will work.

Reply:
Okay, thanks.